vector/src/main/java/im/vector/app/features/home/room/detail/TimelineViewModel.kt

```diff
@@ -266,6 +278,7 @@ class TimelineViewModel @AssistedInject constructor(
     }
 
     private fun prepareForEncryption() {
+        if (room == null) return
```

Comment:
I could have used `?` like `room?.` and `timeline?.` everywhere, but I think this is cleaner like this. When the room is null, there is nothing we can really do with this ViewModel.